### PR TITLE
form validation check

### PR DESCRIPTION
**Pull request by HeDevedUp**
displaying form fields for the contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -175,13 +175,52 @@ <h1>Skills</h1>
...
+                    labore et dolore magna.</p>
+            </section>
+            <form action="https://formspree.io/f/xknyprke" method="POST">
+                <ul>
 
+                    <li> <input class=" name-input" name="text" type="text" placeholder="Name">
+                    </li>
+                    <li> <input class="mail-input" name="email" type="email" placeholder="EMAIL"> </li>
+                    <li><textarea class="textarea-input" name="textarea" id="textarea" cols="30"
+                            rows="10"> hello </textarea></li>
```
Comment (MeqdamAlqudah) [inline index.html]: - [x] Please use the same design as in the Figma design 

| Your design | Figma |
| ------------- | ------- |
| image | image |
| 1-notice how you don't have the same image in the corner as in Figma design,2- please make sure that the placeholder text is the same as in Figma design | |

Comment (MeqdamAlqudah) [inline index.html]: - [x] Please make sure to Implement the following: - 
1. Name input has `maxlength="30"` attribute.
1. Message input has `maxlength="500"` attribute.
1. Email input's `type` equals `email`.
2. All input fields have the `required` attribute to activate the validation on submit.

File: index.html [hunk 2]
```diff
@@ -175,13 +175,52 @@ <h1>Skills</h1>
...
 
-            <div></div>
+                </ul>
+                <button type="submit" class="btn profilelin">
+                    <span class="toprofile">Start collaboration</span>
+                </button>
```
Comment (MeqdamAlqudah) [inline index.html]: - [ ] Please fix this weird distance make sure you have a responsive design 

image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @officailesse , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (sja-thedude) [reply index.html]: Fixed good job 👍

File: index.html [hunk 3]
```diff
@@ -175,13 +175,83 @@ <h1>Skills</h1>
...
+                    adipiscing elit, sed do eiusmod tempor incididunt ut
+                    labore et dolore magna.</p>
+            </div>
+
+
+            <form action="https://formspree.io/f/xknyprke" method="POST">
+
+                <div class="form-container">
+                    <label for="name"></label>
+                    <input type="text" id="name" name="name" placeholder="name" required />
```
Comment (sja-thedude) [inline index.html]: - [x] As suggested by the previous reviewer, kindly add maxlength="30" attribute to the Name input.

File: index.html [hunk 4]
```diff
@@ -175,13 +175,83 @@ <h1>Skills</h1>
...
+            </form>
         </div>
     </section>
+
+    <footer class="personal-contact">
+        <div class="resume-con">
+            <p class="link-icon">Get my Resume</p>
+            <a href="/index.html"> <img class="link-image" src="./assert/ic_link_teal.png"
+                    alt="An simbol that work as link"></a>
+        </div>
```
Comment (sja-thedude) [inline index.html]: - [ ] As suggested by the previous reviewer, Kindly fix the background image at the top left as shown in the below video.
- [ ] Optional: It is highly recommended to implement this, Kindly make sure to align all the input fields with the same width as that of the Figma design.
- [ ]  As suggested by the previous reviewer kindly make sure that the placeholder text for the name is the same as in Figma design as "Your name".
- [ ] Kindly make sure that the button (Start Collaboration) is visible without hovering.
- [ ] Optional: It is highly recommended to implement this, Kindly add all the social media Icons as given in the Figma design.

https://user-images.githubusercontent.com/70269507/165522882-3f6f4059-855d-4fed-908f-798bde5d8c9b.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @officailesse, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job in implementing the change so far.

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**